ConnorAndersonLarson/Intention-Timer
# Feature/logging past activities

## Is this a fix or a feature?

This is a feature

## What is the change?

Added save functionality to past activities. When returning to the page and past activities on the right side will remain.  Made sure timer circle changes color after creating a new activity.

## What does it fix?

No local storage previously present. Timer circle would not change color after first activity.

## Where should the reviewer start?

Most changes were done in main.js, namely saveActivities, loadActivities,  showActivity, and changeCircleCover.  
In activity.js the countdown method was lightly altered and the saveToStorage was fleshed out.

## How should this be tested?

Pull this repo/branch. Run multiple activities, noting the color of the circle each time. After a few activities are completed refresh the site and check that the past activities re-appear and are in the proper order.
